src/Xamarin.Android.Build.Tasks/Utilities/ZipArchiveEx.cs

```diff
@@ -27,8 +31,10 @@ public ZipArchiveEx(string archive, FileMode filemode)
 			zip = ZipArchive.Open(archive, filemode);
 		}
 
-		public void Flush ()
+		public void Flush (bool forceFlush = false)
```

Comment:
I don't understand this change.   If someone is calling `.Flush()`, why *shouldn't* we just flush?